supertokens_python/recipe/session/interfaces.py

```diff
@@ -304,15 +308,6 @@ async def update_session_data_in_database(
     ) -> bool:
         pass
```

Comment:
the sessionContainer interface doesn't have `get_all_session_tokens_dangerously` defined. Please define it.

Make sure that the session container interface is exactly the same as in the node SDK - some functions have been removed, some have been added.. some have been changed in name, and some args have changed. No mistakes please.

Reply:
Done.